web-share/share-extra-argument-manual.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Surplus arguments ignored</title>
```

Comment:
I think it makes sense to test this.

Surely if we extend the API later, we'll also extend the test? More importantly, this tests that a valid implementation *allows* us to extend the API later without breaking backwards-compatibility.

(The tests don't need to be forwards-compatible, but we do need to test that the implementation is forwards-compatible.)